README.md

```diff
@@ -127,8 +131,11 @@ This will likely involve DNS or a hosts entry as well as a webserver
 
 #### Run Janus
 
-Use sbt to run Janus in development mode. The server will
-automatically recompile and reload when changes are made.
+Use sbt to run Janus in development mode. 
+
+The Node bundle is built using a `PlayRunHook`.
```

Comment:
Can we link to the run hook here, to help people find it? I'd consider linking to the run hook class and mentioning the settings in the build.sbt, to help people see how this all fits together (since it is quite unusual).

Not urgent!